lib/external/sentry.js

```diff
@@ -1,7 +1,92 @@
 const { inspect } = require('util');
 const { isBlank } = require('../util/util');
 
-const sensitiveEndpoints = ['/users/reset/verify', '/users/(.*?)/password'];
+
+// Endpoints where query strings are sensitive and should be removed
+const sensitiveEndpoints = [
+  ['GET', '/users[^/]'], // ?q=<part of name or email>
```

Comment:
Does `[^/]` need to be in there? I think it'll just match the `?` of the query string?

Reply:
Oh I see, it's there in order not to match endpoints like `/v1/users/reset/initiate`. :+1: